sashimi/state.py

```diff
@@ -472,77 +274,18 @@ def send_camera_settings(self):
 
     def send_scan_settings(self, param_changed=None):
         # Restart scanning loop if scanning params have changed:
-        if self.global_state == GlobalState.VOLUME_PREVIEW:
+        if self.global_state == ScanningUseMode.VOLUME:
             self.restart_event.set()
 
-        # # Sync piezo position across modalities, for usability:
```

Comment:
@vigji why is this all seemingly good-looking code all commented (not really related to this PR but I am eager to know)